spec/requests/api/v1/show_people_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'rails_helper'
```

Comment:
Do you have controller called ShowPeople ? if not. I would combine all test for giving controller under one file with controller name.

Reply:
@ssharba  please read this may help you! 
 
 ActionController::TestCase is deprecated
(https://blog.bigbinary.com/2016/04/19/changes-to-test-controllers-in-rails-5.html)


While controller tests themselves haven’t been removed in Rails 5, a couple of commonly-used helpers, assigns and assert_template, have been deprecated
(https://everydayrails.com/2016/08/29/replace-rspec-controller-tests.html)  


It allows you to simulate a single http request in each example, and then
specify expected outcomes such as:
-rendered templates
-redirects
-instance variables assigned in the controller to be shared with the view
-cookies sent back with the response
(https://relishapp.com/rspec/rspec-rails/docs/controller-specs)

Reply:
@louy755 I still want you to change the name of the test to reflect controller or model name.